py_ocpi/tokens/v_2_2_1/api/cpo.py

```diff
@@ -1,10 +1,73 @@
-from fastapi import APIRouter
+from email.policy import default
+from fastapi import APIRouter, Request, Depends, Query
+from pydantic import ValidationError
+
+from py_ocpi.core import status
+from py_ocpi.core.data_types import CiString
+from py_ocpi.core.enums import ModuleID
+from py_ocpi.core.schemas import OCPIResponse
+from py_ocpi.core.utils import get_auth_token
+from py_ocpi.core.dependencies import get_crud, get_adapter
+from py_ocpi.tokens.v_2_2_1.enums import TokenType
+from py_ocpi.tokens.v_2_2_1.schemas import Token, TokenUpdate
 
 router = APIRouter(
     prefix='/tokens',
 )
 
 
-@router.get("/")
-def get_tokens():
-    return "tokens app created!"
+@router.get("/{country_code}/{party_id}/{token_uid}", response_model=OCPIResponse)
+async def get_token(country_code: CiString(2), party_id: CiString(3), token_uid: CiString(36),
+                    request: Request, crud=Depends(get_crud), adapter=Depends(get_adapter)):
```

Comment:
Optional type parameter also should be provided as in other methods (according to docs)